test/io/stonea/printDifferentRankArrays.chpl

```diff
@@ -1,8 +1,12 @@
+use IO, ChplFormat;
+
 var A: [1..2] string = "hi";
 var B: [1..2, 1..2] string = "hi";
 
 writeln(A);
 writeln(B);
 
-writef("%ht\n", A);
-writef("%ht\n", B);
+var chplOut = stdout.withSerializer(ChplSerializer);
```

Comment:
I think this would be good to test with the default and JSON serializer as well

Reply:
That functionality does get tested by `io/serializers/arrays.chpl`